infrastructure/preview.tfvars

```diff
@@ -1,3 +1,3 @@
-idam_api_url = "https://preprod-idamapi.reform.hmcts.net:3511"
-#idam_api_url = "https://idam-api.aat.platform.hmcts.net"
-frontend_url = "https://www-ccd.nonprod.platform.hmcts.net"
+#idam_api_url = "https://preprod-idamapi.reform.hmcts.net:3511"
+idam_api_url = "https://idam-api.aat.platform.hmcts.net"
+frontend_url = "https://www-ccd.aat.platform.hmcts.net"
```

Comment:
We technically don't need this one IIUC.  But not a problem.

Reply:
👍